joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -916,7 +916,7 @@ TEST_F(TestTrajectoryController, test_trajectory_replace) {
   subscribeToState();
 
   std::vector<std::vector<double>> points_old {{{2., 3., 4.}}};
```

Comment:
how come it doesn't complain about this line for the same reason? can you try changing this too for consistency pelase?

Reply:
it does not. I think there's two different constructors being called.

Reply:
It probably is the case. Let's keep an eye out if something else puts it back again!

![3hpxy5](https://user-images.githubusercontent.com/3524577/95909726-e13c1a00-0d96-11eb-8b22-2d0836b974b4.jpg)